README.md

```diff
@@ -34,7 +33,7 @@ After installing you can run:
 > http-api-docs
 ```
 
-This should spit out a Markdown document. This is exactly the `api.md` documentation at https://github.com/ipfs/docs/blob/master/content/reference/api/http.md, so you can redirect the output to just overwrite that file.
+This should spit out a Markdown document. This is exactly the `api.md` documentation at https://github.com/TRON-US/go-btfs/tree/master/docs/content/reference/api/http.md, so you can redirect the output to just overwrite that file.
```

Comment:
Update LICENSE section on the bottom as well.

Reply:
Also the LICENSE file itself.

Reply:
@taiyangc not really sure what to update here...

Reply:
Could you change `MIT (C) Protocol Labs, Inc.` to `MIT (C) TRON-US`? similarly in the LICENSE file

Reply:
ok.